### PR TITLE
Setup gitlab

### PR DESCRIPTION
**Pull request by jacobwindsor**
For [#257](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/257)

- Prepares the helm charts for integration with Gitlab by adding the gitlab annotation
- Removes image refs as we won't be updating this anymore. Only gitlab will be deploying 
- Changes deploy-app command to accept an image tag
- Add deploy-ontology which uses the image tag in the configs

### CODE REVIEW

**Review by aaclan-ebi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -112,16 +112,21 @@ For example, in dev, the CIDR is set to `10.40.0.0/16`. The Terraform manifest a
 Coming soon
 
 ## Deploy and Upgrade Ingest Applications
+Deployments are automatically handled by [Gitlab](https://gitlab.ebi.ac.uk/) but you can still manually deploy if required (see below). However, `ontology` is not deployed by Gitlab but there is a special command for deploying ontology (see below).
 
-### Deploy one kubernetes dockerized applications to an environment (aws)
+### Manually deploy one kubernetes dockerized applications to an environment (aws)
```
Comment (aaclan-ebi) [inline README.md:117]: All of these are manual haha

Comment (jacobwindsor) [reply README.md:117]: Just emphasizing that we should only do this if we have to

Comment (aaclan-ebi) [reply README.md:117]: I see. You could put a note below it instead. That deployment are being done via gitlab and should avoid manual running of these commands unless necessary.

Comment (jacobwindsor) [reply README.md:117]: There is a note above

Comment (aaclan-ebi) [reply README.md:117]: oops, has that been there before? my bad! haha okidokies!

**Pull request merged**